lib/assets/javascripts/bootstrap_listeners.js

```diff
@@ -15,9 +15,10 @@ $(document).ready(function(){
         $(this).parent().find('.minus-laranja').removeClass('minus-laranja').addClass('plus-laranja');
     });
     // Initialises all tooltips present on a page
-    $('.has-tooltip').tooltip({
-        placement: "right",
-        trigger: "focus"
-    });
+    $('.has-tooltip').tooltip({ placement: "right", trigger: "focus" });
     $(".help").popover();
+    //Initiliases all popovers on a page
```

Comment:
looks good @jollopre we'll have to remember to choose this one and remove mine when we merge over the CDL-MVP branch. I solved it in a similar way but used `$("a[rel='popover'])`. Yours is better since it doesn't force the use of an anchor